bindings_node/src/identity.rs

```diff
@@ -0,0 +1,65 @@
+use crate::ErrorWrapper;
+use napi_derive::napi;
+use xmtp_cryptography::signature::IdentifierValidationError;
+use xmtp_id::associations::{ident, PublicIdentifier as XMTPPublicIdentifier};
+
+#[napi(object)]
+#[derive(Clone, Hash, PartialEq, Eq)]
+pub struct PublicIdentifier {
+  pub identifier: String,
+  pub identifier_kind: PublicIdentifierKind,
+  pub relying_partner: Option<String>,
+}
+
+#[napi]
+#[derive(Hash, PartialEq, Eq)]
+pub enum PublicIdentifierKind {
+  Ethereum,
+  Passkey,
+  // more to come...
```

Comment:
do we need this anymore? or just a placeholder?

Reply:
I mean, it's just a comment that hints at the future inclusion of other identifiers. I can remove it if it's confusing.